pretty_cron/api.py

```diff
@@ -117,15 +142,29 @@ def _human_week_day(day):
 
 
 def _ordinal(n):
-    if 10 <= (n % 100) < 20:
-        suffix = 'th'
+    """
+    Cron-tab to numeric day
+    """
+    if isinstance(n, tuple):
```

Comment:
OMG, yeah. Thanks. I guess we also could apply this change at _human_month and _human_week_day?